src/sequences/OEIS.ts

```diff
@@ -129,29 +140,26 @@ export default class OEIS extends Cached {
         }
     }
 
-    checkParameters(): ValidationStatus {
-        const status = super.checkParameters()
+    checkParameters(params: ParamValues<typeof paramDesc>): ValidationStatus {
+        const status = super.checkParameters(params)
 
         if (
-            this.params.oeisId.value.length !== 7
-            || (this.params.oeisId.value[0] !== 'A'
-                && this.params.oeisId.value[0] !== 'a')
+            params.oeisId.length !== 7
+            || (params.oeisId[0] !== 'A' && params.oeisId[0] !== 'a')
         ) {
-            status.errors.push('OEIS IDs are of form Annnnnn')
+            status.addError('OEIS IDs are of form Annnnnn')
         }
-        if (typeof this.params.cacheBlock.value === 'number') {
+        if (typeof params.cacheBlock === 'number') {
```

Comment:
The comment here is similar in spirit to the other new one about `as number`, but not just explicitly another instance of that exact same thing. As far as I can see, with the new param system, it's now logically certain that params.cacheBlock wiil be of type number, and so this check can be removed. If you agree, please go ahead and remove the check. If you disagree, please let us know a scenario in which params.cacheBlock won't be a number when the code gets to this point, so we can decide how the situation should be handled, Thanks!